StPetersburg/InEtn660715.xml

```diff
@@ -0,0 +1,268 @@
+<?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="InEtn660715" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Collection of magic texts</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens: Eine multimediale
+                            Forschungsumgebung / beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+              <msDesc xml:id="ms">
+                <msIdentifier>
+                    <repository ref="INS0524InEtn"/>
+                    <idno>6607-15</idno>
+                    <altIdentifier>
+                      <idno>Platonov cat. III.131</idno>
+                    </altIdentifier>
+                </msIdentifier>
+                <msContents>
+                  <summary>
+                  </summary>
+                  <msItem xml:id="ms_i1">
+                    <locus from="1r" n="1"></locus>
+                    <locus to="1r" n="13"></locus>
+                    <title type="complete" ref="LIT5889UndoingOfCharms" xml:lang="gez">መፍትሔ፡ ሥራይ፡</title>
+                    <textLang mainLang="gez"></textLang>
+                  </msItem>
+                  <msItem xml:id="ms_i2">
+                    <locus from="1r" n="14"></locus>
+                    <locus to="1r" n="38"></locus>
+                    <title type="incomplete" ref="LIT2711Mark"></title>
+                    <textLang mainLang="gez"></textLang>
+                    <note>1.23-28.</note>
+                  </msItem>
+                  <msItem xml:id="ms_i3">
+                    <locus from="1r" n="39"></locus>
+                    <locus to="1r" next="52"></locus>
+                    <title type="complete" ref="LIT5889UndoingOfCharms"></title>
```

Comment:
See also above. Some disambiguation is also needed between LIT5127MagicPrayer (no text is provided there) and other Prayers of the same title

Reply:
I would be happy to do that, but the catalogue provides NO text, so no kind of disambiguation can be done. That's ehy this general record was created.

Reply:
LIT1824Mafteh  (used e.g in BLadd16243) was a general record for unspecified Maftehe seray I assumed. 
Unless we can disambiguate (e.g. say that LIT5889UndoingOfCharms is for single short prayers (?) and LIT1824Mafteh for longer collections) the two IDs must be merged.

Reply:
It is in fact like that, LIT1824Mafteh is for the collection of magic prayers which is called Maftehe Seray (the text occupies many folia and has more or less stable content). LIT5889UndoingOfCharms is for single prayers for undoing of charms (no clue whether it belongs to the collection LIT1824Mafteh or not), there is already a number of such identified prayers in the database.

Reply:
Someone should take a good witness of Mäftahe seray and write out part-prayers.

Reply:
Once we are sure that this witness is really representative :)

Reply:
Well, there is always a certain risk that it is not. Otherwise we can wait untill someone will prepare a critical edition.

Reply:
#https://github.com/BetaMasaheft/Works/pull/112